assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -249,6 +313,47 @@ def _update_conda_dependencies(self, package_details):
             yaml.safe_dump(conda_dict, f)
             logger.info("updated conda.yaml")
 
+    def _get_curated_environment_pip_package_list(self) -> List[str]:
+        """
+        Retrieve the packages using 'conda list' command.
+
+        :return: A List of the pip package and the corresponding versions.
+        """
+        import subprocess
+        import json
+
+        PIP_LIST = ['accelerate', 'cffi', 'dill', 'google-api-core', 'numpy',
+                    'packaging', 'pillow', 'protobuf', 'pyyaml', 'requests', 'scikit-learn',
+                    'scipy', 'sentencepiece', 'torch', 'mlflow']
+        ADD_PACKAGE_LIST = ['torchvision==0.14.1', 'transformers==4.34.1']
+
+        if self._task == SupportedTasks.AUTOMATIC_SPEECH_RECOGNITION.value:
+            ADD_PACKAGE_LIST.append(['soundfile==0.12.1', 'librosa==0.10.1', 'diffusers==0.21.4',
```

Comment:
There is a conda.yaml for whisper model, since it has hardcoded version, we dont wanna use it?